text/0000-channel-participation-api-without-system-channel.md

```diff
@@ -0,0 +1,497 @@
+---
+layout: default
+title: Channel participation api without system channel
+nav_order: 3
+---
+
+- Feature Name: channel_participation_api_without_system_channel
+- Start Date: 2020-03-01
+- RFC PR: (leave this empty)
+- Fabric Component: orderer
+- Fabric Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Currently in order to create a channel one needs to issue a config transaction to the system channel. This has 
+privacy, scalability and operational disadvantages, since the system channel (and the orderers running it) is aware of 
+all channels and of all channel members (at creation time). In this feature we propose to expose a 
+"channel participation" API that would allow a local orderer administrator to join and leave a channel, as well as 
+to list all the channels that the local orderer is part of. This allows a Fabric network to be operated without the use 
+of a system channel, which improve the privacy and scalability of the ordering service and the network as a whole. 
+
+
+# Motivation
+[motivation]: #motivation
+
+Creating channels by issuing a transaction to the system channel presents several disadvantages.
+
+## Privacy problems
+A system channel requires that all ordering service nodes (OSNs) be aware of all channels. As Fabric depricated the 
+Kafka-based ordering service and moved to Raft (and later is planned to introduce BFT consensus), it is no longer 
```

Comment:
```suggestion
Kafka-based ordering service and moved to Raft, it is no longer 
```

No need to mention any future stuff here IMHO

Reply:
Fixed